File: app/src/main/java/com/juniperphoton/myerlist/activity/BaseActivity.java
package com.juniperphoton.myerlist.activity;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import com.juniperphoton.myerlist.util.StatusBarCompat;

public class BaseActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        StatusBarCompat.setUpActivity(this);
        //getWindow().setFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS, WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS);
    }
}

File: app/src/main/java/com/juniperphoton/myerlist/activity/BaseActivity.kt
package com.juniperphoton.myerlist.activity

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.juniperphoton.myerlist.R
import com.juniperphoton.myerlist.util.LocalSettingUtil

import com.juniperphoton.myerlist.util.StatusBarCompat
import java.util.*

open class BaseActivity : AppCompatActivity() {
    @Suppress("deprecation")
    override fun onCreate(savedInstanceState: Bundle?) {
        val languageKey = resources.getString(R.string.language_key)
        val language = when (LocalSettingUtil.getInt(this, languageKey, 0)) {
            0 -> Locale.ENGLISH
            else -> Locale.SIMPLIFIED_CHINESE
        }
        val config = resources.configuration
        config.setLocale(language)
        resources.updateConfiguration(config, resources.displayMetrics)

        super.onCreate(savedInstanceState)
        StatusBarCompat.setUpActivity(this)
    }
}